src/workers/notifications/userStreakResetNotification.ts

```diff
@@ -32,7 +32,7 @@ const worker = generateTypedNotificationWorker<'api.v1.user-streak-updated'>({
       streak: {
         ...streak,
         currentStreak: parseInt(lastStreak, 10),
-        lastViewAt: new Date(streak.lastViewAt!).getTime(),
+        lastViewAt: new Date(streak.lastViewAt!),
```

Comment:
@sshanzel, for sanity check, strict type requires Date, from what I see in notification handler `format` accepts `Date | number | string` so it should be good?

Reply:
test also fails for it because it expects number https://app.circleci.com/pipelines/github/dailydotdev/daily-api/9926/workflows/6d3c9906-2592-4ff0-a78d-33267a4a1751/jobs/39988/tests

Reply:
I vaguely remember if there was an issue with `format` accepting the value. But if it works, then yeah, we should be good here too.

Reply:
Maybe we can manually update the expected value from int to string since that is what will happen now.